README.md

```diff
@@ -1,50 +1,69 @@
 # BigchainDB Whitepaper
 
+![repo banner](media/repo-banner@2x.png)
+
 This repository contains the source (LaTeX) for the BigchainDB whitepaper.
 
+---
+
+**The most recent PDF version of the whitepaper can always be retrieved from [bigchaindb.com/whitepaper](https://www.bigchaindb.com/whitepaper)**
+
+---
+
 If you're looking for the main BigchainDB repository, it's at [bigchaindb/bigchaindb](https://github.com/bigchaindb/bigchaindb).
 
+And if you're really here to get a glimpse of the exciting world of LaTeX-based file generation, this repo is for you. 
 
-## Prerequisites for Using Bash Script or Make
+## Prerequisites for generating the PDF
 
-1. [TeX Live](https://www.tug.org/texlive/) or similar
-2. pdftk (can be installed using `apt-get install pdftk`)
+Build requires two binaries:
 
+- `pdflatex`
+- `bibtex`
 
-## Using the Bash Script to Generate the PDF
+### Using Docker
```

Comment:
In general: all documentation should present a preferred way of doing things instead of listing multiple options and leaving it to user to chose from. Since it's almost impossible to get the requirements running (except on an OS with ~1% market share), the Docker way should be the preferred way. On Mac & Windows, if you have Docker installed, you have Docker Compose too.

But you're right, it's actually no prerequisite, but as outlined above I would move it to the top, not the end.